### PR TITLE
[Blog] DockerCon 2022 Talk with HashiCorp | CU-2g4fptm

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
Talk 1 with HashiCorp DC22

This PR fixes # CU-2g4fptm

**Notes for Reviewers**
MERGE on the 10th evening DockerCon 2022 only,

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
Task linked: [CU-2g4fptm DC22: Blog post recap of talk with HashiCorp](https://app.clickup.com/t/2g4fptm)

**Comment by l5io** [comment]
🚀 Preview for commit ad7e740c47a74519375899670e85d742da776a82 at: https://6276fe6705e5fd52e47e6fa0--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Thank you

**Comment by Nikhil-Ladha** [comment]
Time to merge?

**Comment by Chadha93** [comment]
> Time to merge?

Nope, this was discussed yesterday, I'll update some minor content and will attach the video link of DockerCon session with the transcript tonight/tomorrow, then we can publish.

**Review by leecalcote — APPROVED** [review]

File: src/collections/blog/2022-05-07-dockercon-22-hashicorp-talk/index.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import dashboard from './dashboard.png';
+import designer from './designer-1.png';
+import visualizer from './viz-1.png';
+import thankyou from './thankyou.png';
+
+<BlogWrapper>
+
+<div class="intro ">
+<p>
+    <Link to="/community/members/lee-calcote">Lee Calcote</Link> and <Link to="/community/members/nic-jackson"> Nic Jackson </Link> gave a presentation entitled Intro and Deep Dive into Docker Extension for Meshery at DockerCon 2022.
```
Comment (leecalcote) [inline src/collections/blog/2022-05-07-dockercon-22-hashicorp-talk/index.mdx:30]:
```suggestion
    <Link to="/community/members/lee-calcote">Lee Calcote</Link> and <Link to="/community/members/nic-jackson">Nic Jackson</Link> gave a presentation entitled <i>Extending the Docker Compose Experience to Service Mesh</i> at DockerCon 2022.
```

File: src/collections/blog/2022-05-07-dockercon-22-hashicorp-talk/index.mdx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+</p>
+</div>
+
+<p>
+    The Docker Extension for Meshery extends Docker Desktop's position as the cloud native developer's go-to Kubernetes environment with easy access to the next
```
Comment (leecalcote) [inline src/collections/blog/2022-05-07-dockercon-22-hashicorp-talk/index.mdx:35]:
```suggestion
    The <Link to="/docker-extension-meshery">Meshery Docker Extension</Link> extends Docker Desktop's position as the cloud native developer's go-to Kubernetes environment with easy access to the next
```

**Comment by leecalcote** [comment]
@gauravchauhan2403 incorporate this link, please - https://www.docker.com/blog/docker-captain-take-5-lee-calcote/

See this example - https://layer5.io/company/news/layer5-joins-docker-extension-program-bringing-kubernetes-and-service-mesh-management-to-docker-with-meshery

File: src/collections/blog/2022-05-07-dockercon-22-hashicorp-talk/index.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+<p>
+    To Lee's demo, Nic also added in the context of developer experience that MeshMap also makes it easy to do the job and with Meshery extension it makes it super easy to write and configure code without being worried for the developer environment.
+    </p>
+    <p>
+       These Docker Extensions are so powerful that it allows you to do multiple tasks without leaving Docker Desktop. 
+        </p>
+
+<p><strong>Lee Calcote and Nic Jackson packed a great deal of information in this talk. The Meshery Extension is now out! Try now, and Share your Experience </strong>
+<a href="https://layer5.io/service-mesh-management/meshmap">Apply for MeshMap Beta Program</a>
```
Comment (leecalcote) [inline src/collections/blog/2022-05-07-dockercon-22-hashicorp-talk/index.mdx:112]:
```suggestion
<Link to="/service-mesh-management/meshmap">Apply for MeshMap Beta Program</Link>
```

**Comment by l5io** [comment]
🚀 Preview for commit 3a50aeed598280c705989e7ee1133ee2c902b622 at: https://627a9307b5a19c48d0b4aebe--layer5ng.netlify.app

**Pull request merged**